### PR TITLE
[Mailer] Add Support for email-validator v3

### PR DESCRIPTION
**Pull request by ben29**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | yes
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by michaljusiega** [comment]
Ticket: https://github.com/symfony/symfony/issues/39656

**Comment by OskarStark** [comment]
Can please check if other Components use this too and bump them also? Thanks

File: src/Symfony/Component/Mailer/composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "egulias/email-validator": "^2.1.10",
+        "egulias/email-validator": "^2.1.10|^3.0",
```
Comment (derrabus) [inline src/Symfony/Component/Mailer/composer.json:20]: There are more places where this package needs to be bumped. Furthermore, we should try to patch the 4.4 branch instead.

**Review by derrabus — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Mailer/composer.json (hunk 1 above)
Comment (ben29) [reply src/Symfony/Component/Mailer/composer.json:20]: yes, on all the mailer components

Comment (OskarStark) [reply src/Symfony/Component/Mailer/composer.json:20]: I would prefer 4.4, too

**Comment by derrabus** [comment]
Closing in favor of #39685.

**Pull request closed without merging**